src/db/logger.cr

```diff
@@ -0,0 +1,30 @@
+require "logger"
+
+module DB
+  # :nodoc:
+  class Logger
+    @logger : Proc(String, Void)?
+    @enabled = false
+    @default_logger = ::Logger.new(STDOUT)
+
+    def config(@logger)
+    end
+
+    def logging=(@enabled : Bool)
+    end
+
+    def log(query : String, *args)
+      return if !@enabled
+      if args && args.size > 0
+        log = %{QUERY "#{query}" WITH PARAMS: #{args.join(", ")}}
```

Comment:
I agree the logging could be a little better, but I quite like it stating with QUERY : as it stands out amongst other log entries. Within kemal, the default log follows this format with the http verb being in block caps.
```
2017-07-25 21:25:07 +0100 200 GET /duocms/api/groups 16.34ms
2017-07-25 21:25:07 +0100 QUERY "SELECT `id`,`name`,`order` FROM groups where deleted_at is null and id = ?" with params: 1
```